dotcom-rendering/src/components/FeatureCard.tsx

```diff
@@ -125,9 +133,21 @@ const overlayStyles = css`
 	flex-direction: column;
 	justify-content: flex-start;
 	flex-grow: 1;
-	padding: ${space[2]}px;
 	gap: ${space[1]}px;
-	backdrop-filter: blur(12px) brightness(0.7);
+	padding: 64px ${space[2]}px ${space[2]}px;
```

Comment:
I've left this as the raw value rather than using `space[16]` from Source as it ties into the colour stops in the gradient below.